src/Compilers/CSharp/Portable/Symbols/Source/GlobalExpressionVariable.cs

```diff
@@ -50,7 +50,7 @@ internal class GlobalExpressionVariable : SourceMemberFieldSymbol
             Debug.Assert(nodeToBind.Kind() == SyntaxKind.VariableDeclarator || nodeToBind is ExpressionSyntax);
 
             var syntaxReference = syntax.GetReference();
-            return (typeSyntax == null || typeSyntax.SkipScoped(out _).SkipRef(out _).IsVar)
+            return (typeSyntax == null || typeSyntax.SkipScoped(out _).SkipRef().IsVar)
```

Comment:
sidenote: we have a ton of code taht does this unwrapping to check .IsVar.  We may want to consider having that property just do this unwinding anyways.